composer.json

```diff
@@ -29,13 +29,11 @@
         "natxet/cssmin": "~3.0.0",
         "php": "^7.1",
         "phpmailer/phpmailer": "~6.0",
-        "sensio/distribution-bundle": "^5.0.19",
-        "symfony/monolog-bundle": "^3.1.0",
-        "symfony/polyfill-apcu": "^1.0",
-        "symfony/symfony": "~3.4.35",
-        "symfony-cmf/routing-bundle": "~2.1.0",
-        "tedivm/jshrink": "~1.0",
-        "twig/twig": "~2.8"
+        "symfony/monolog-bundle": "3.5.0",
+        "symfony/polyfill-apcu": "1.13.1",
+        "symfony/symfony": "4.4.4",
```

Comment:
Should this version be fixed?

Reply:
This was done intentionally because at least on my machine I couldn't get composer to resolve the massive dependency graph using less strict versions. We should address this in a future update, but it is not necessarily bad practice to define exact versions - if you take care of them from time to time.

Reply:
Ok, should be done with a "broader" version.

But working / testing with this is easier with this merged...